test/cases/aws.sh

```diff
@@ -263,6 +263,12 @@ for LOOP_COUNTER in {0..10}; do
     sleep 5
 done
 
+# Ensure the image was properly tagged.
+IMAGE_TAG=$($AWS_CMD ec2 describe-images --image-ids "${AMI_IMAGE_ID}" jq -r '.Images[0].Tags[] | select(.Key=="Name") | .Value')
```

Comment:
```suggestion
IMAGE_TAG=$($AWS_CMD ec2 describe-images --image-ids "${AMI_IMAGE_ID}" | jq -r '.Images[0].Tags[] | select(.Key=="Name") | .Value')
```

You lost a pipe.

![image](https://user-images.githubusercontent.com/119861/105477512-39da5780-5ca2-11eb-80e5-eb3da448e195.png)

Reply:
Oopsies. It should be fixed now. 👨🏻‍🔧